### DISCUSSION TITLE
Property "required" not working

### DISCUSSION THREAD

**Discussion by Shepych** [DarkaOnLine/L5-Swagger#534] [Q&A]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103124683/226999375-fab200cd-a1d4-4c2f-9ca8-302c1d0e92c0.png">
If attributes "required" are specified, then an error JS (index.js, system.js) occurs when executing the request. And the infinite preloader "Loading" appears
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103124683/226999951-74f1a9e6-8718-4f65-b19a-59b9f1eab679.png">
<img width="862" alt="image" src="https://user-images.githubusercontent.com/103124683/227000901-18e757a7-464d-4247-93f6-b1aa61e5fcff.png">

**Comment by Shepych** [comment]
GG

**Comment by DarkaOnLine** [comment]
Could you give full `@OA/Post` annotation? From screenshot it looks like incomplete or invalid

**Reply by Shepych** [reply]
![]( https://user-images.githubusercontent.com/103124683/227187155-e137bba1-bda0-4fee-aed5-405a5138350c.png)
@OA\Get has also this bug

**Reply by DarkaOnLine** [reply]
At the end next to closing `)` for `@OA/Parameter` you have comma `,` -> `),`.  This should be just `)`. Could this be an issue? 

```
     /**
     * @OA\Get(
     *     ...
     *     @OA\Parameter(
     *         name="id",
     *         description="ID of pet that needs to be fetched",
     *         required=true
     *     )
     * )
     * /
```

This is an example. Everything should be working: https://github.com/zircote/swagger-php/blob/master/Examples/petstore-3.0/Controllers/Store.php#L73-L84

**Comment by Shepych** [comment]
Yes, but this not working in PHP Framework Laravel

**Reply by DarkaOnLine** [reply]
What do you mean? Annotations are not depending on framework

**Comment by Shepych** [comment]
Set Laravel and check this)) In laravel v8, v9, v10 has this bug ... I dont know cause

**Comment by Shepych** [comment]
And when the project is loaded on production - the link /api/documentation does not work due to the fact that it cannot load the assets css and js. I had to manually create these files for him to see them, there may be a problem with the routes / file locations, but property "required" anyway still didn't work
